reader/readerContent.css

```diff
@@ -98,13 +99,14 @@ ol li {
 	line-height: 1.5em;
 }
 a {
-	color: #1b74bb;
+	color: #1e90ff;
 	font-size: 1em;
 	text-decoration: none;
 }
 body[theme="dark"] a {
-	color: #65a9ed;
+	color: #1e90ff;
```

Comment:
Can you switch back to the original colors? In a paragraph with a lot of links, the old colors made it easier to read the paragraph without looking at the links:

<img width="700" src="https://user-images.githubusercontent.com/10314059/82606491-c9ac8c80-9b7c-11ea-820a-cfb2308cf18a.png">

But with these colors, the links stand out more than the rest of the text, and it's hard to read:

<img width="723" src="https://user-images.githubusercontent.com/10314059/82606585-ef399600-9b7c-11ea-918e-97e157d9cb5f.png">

Reply:
adjusted